### PR TITLE
Math Magicians: Tests

### PR DESCRIPTION
**Pull request by rodrigopenayo98**
- Set up React Testing Library.
- Write unit tests for the files operate.js and calculate.js using Jest.
- Create unit tests for all React components.
- All tests passed.

### CODE REVIEW

File: src/components/Home.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (DammyShittu) [inline src/components/Home.test.js]: - [ ] Kindly add snapshot tests for ALL your components.

File: src/components/Home.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import { render, screen } from '@testing-library/react';
```
Comment (DammyShittu) [inline src/components/Home.test.js]: - [ ] Kindly add tests that simulate user actions, like clicking buttons and making calculations using the React testing Library.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @rodrigopenayo98 and @meryemsanem,

## Highlights
✔️ Passing tests

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ All tests are passing

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**